src/AGPL-3.0.xml

```diff
@@ -1,580 +1,853 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <SPDXLicenseCollection xmlns="http://www.spdx.org/license">
-   <license isOsiApproved="true" licenseId="AGPL-3.0"
-            name="GNU Affero General Public License v3.0">
-      <crossRefs>
-         <crossRef>http://www.gnu.org/licenses/agpl.txt</crossRef>
-         <crossRef>http://www.opensource.org/licenses/AGPL-3.0</crossRef>
-      </crossRefs>
-      <standardLicenseHeader> Copyright (C) 
-    <alt match=".+" name="copyright">[year] [name of author]</alt> This program is free software: you can
-    redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the
-    Free Software Foundation, version 3. This program is distributed in the hope that it will be useful, but WITHOUT
-    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-    GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General
-    Public License along with this program. If not, see &lt;http://www.gnu.org/licenses/&gt;
-  </standardLicenseHeader>
-      <notes>This version was released: 19 November 2007</notes>
-      <titleText>
-         <p>GNU AFFERO GENERAL PUBLIC LICENSE 
-        <br/>Version 3, 19 November 2007 
+  <license licenseId="AGPL-3.0" isOsiApproved="true"
+  name="GNU Affero General Public License v3.0"
+  isDeprecated="true" deprecatedVersion="3.0">
+    <crossRefs>
+      <crossRef>http://www.gnu.org/licenses/agpl.txt</crossRef>
+      <crossRef>http://www.opensource.org/licenses/AGPL-3.0</crossRef>
+    </crossRefs>
+    <notes>
+      DEPRECATED: Deprecated in lieu of more
+      explicit license identifier of AGPL-3.0-only
```

Comment:
I copied verbatim the text @jlovejoy used in #545.
Of course I'm fine with changing it.

BTW, there are a couple of licenses now with two distinct `<notes>` elements. I hope this is allowed...

Reply:
The 2 notes element is allowed by the schema.  I just checked and the tool that generates the website is only picking up one of the notes.  Submitted a bug for the tool: https://github.com/spdx/tools/issues/131

I'll try to fix before the release.

Reply:
@jlovejoy Feel free to update or ignore the comment on the notes wording change.

Reply:
> BTW, there are a couple of licenses now with two distinct `<notes>` elements. I hope this is allowed...

I think this is an XSD typo, because the intention [with][1]:

```xml
<element name="notes" type="tns:formattedFixedTextType" minOccurs="0" maxOccurs="1"/>
```

seems to be to only allow a single entry.  I'll file an XSD patch.  I'm fine allowing multiple entries if we use `<note>`, but I think having multiple `<notes>` is strange.

[1]: https://github.com/spdx/license-list-XML/blob/40bf3d9bd56c2415695dacb9f2c64c4bd6fbe29e/schema/ListedLicense.xsd#L41

Reply:
> I'll file an XSD patch

Ah, looks like I already covered this with the `<choice>` → `<all>` change in #452.